src/client/resources/resources/recipes.json

```diff
@@ -142,6 +142,32 @@
       }
     }
   },
+  "minicraft.advancements.recipes.wood_fence": {
+    "criteria": {
+      "has_plank": {
+        "trigger": "inventory_changed",
+        "conditions": {
+          "items": [
+            {
+              "items": [
+                "plank"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "requirements": [
+      [
+        "has_plank"
+      ]
+    ],
```

Comment:
Not quite sure why this is here. I assume this will break the code.

Reply:
It shouldn't break the code as it follows the format